hack/update-gofmt.sh

```diff
@@ -43,5 +43,4 @@ find_files() {
     \) -name '*.go'
 }
 
-GOFMT="gofmt -s -w"
-find_files | xargs "${GOFMT}"
+find_files | xargs gofmt -s -w
```

Comment:
There is similar quote in verify-gofmt.sh
```
diff=$(find_files | xargs "${gofmt}" -d -s 2>&1) || true
```
Should I send out a PR with similar change ?

Reply:
Thanks @tedyu 
But these two are different. 
In `verify-gofmt.sh`, `${gofmt}` is the path of `gofmt`, but not a command.
I don't think we need to modify `verify-gofmt.sh`.